src/wr_logic_ai/nodes/long_range_action_server.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+
+import rospy
+import actionlib
+from wr_logic_ai.msg import LongRangeAction, LongRangeGoal
+from obstacle_avoidance import update_target 
+
+# TODO: check timeout time length validity
+LONG_RANGE_TIMEOUT_TIME = rospy.Duration(5)
```

Comment:
Maybe slightly higher for testing.
```suggestion
LONG_RANGE_TIMEOUT_TIME = rospy.Duration(10)
```

Reply:
Addressed in [277b00a](https://github.com/WisconsinRobotics/WRover_Software/pull/48/commits/277b00a47c8537aba6ba2aeeb6c09d9b5ea4d3eb).